new-dti-website/components/apply/ApplicationTimeline.tsx

```diff
@@ -135,18 +131,16 @@ const TimelineNode: React.FC<RecruitmentEventProps> = ({
           </svg>
         )}
         <div
-          className={`flex items-center justify-center w-[50px] h-[50px] ${
-            isNextEvent
-              ? 'md:w-[70px] md:h-[70px] bg-[#FEFEFE]'
-              : 'md:w-[60px] md:h-[60px] bg-[#F5E3E3] '
-          } rounded-xl md:border-8 xs:border-4 border-solid border-[#A52424D9] z-20`}
+          className={`flex items-center justify-center w-[50px] h-[50px] bg-[#FEFEFE] ${
+            isNextEvent ? 'md:w-[70px] md:h-[70px]' : 'md:w-[60px] md:h-[60px]'
+          } rounded-xl md:border-6 xs:border-4 border-solid border-[#A52424] z-20`}
         >
           <Image
             src={timelineIcons[event.type as keyof typeof timelineIcons].src}
             alt={timelineIcons[event.type as keyof typeof timelineIcons].alt}
-            width={width >= TABLET_BREAKPOINT ? 25 : 20}
-            height={width >= TABLET_BREAKPOINT ? 25 : 20}
-            className={isNextEvent ? 'scale-[1.5] brightness-0' : ''}
+            width={width >= TABLET_BREAKPOINT ? 30 : 25}
```

Comment:
Do you think that maybe we can just keep this static instead of relying on useScreenSize()? Same thing for the height! I feel like the difference is not that noticeable but that might just be me! Wondering what you think and maybe @Bookie0 @andrew032011! We could just set it to be like in the middle like 28:
```suggestion
            width={28} 
```